client/deployment.go

```diff
@@ -20,6 +20,7 @@ type Deployment struct {
 	*RemoteVPS
 	Repository *git.Repository
 	Auth       string
+	Project    string
 }
 
 // DaemonRequester can make HTTP requests to the daemon.
```

Comment:
Do you mind completely removing this interface? it doesn't do much and is tricky to keep up to date. thanks!

Reply:
done

Reply:
Actually @bobheadxi, it turns out to be important for the current solution; I'll try to find other ways that don't rely on this interface.